pkg/api/mlflow/dao/repositories/metric.go

```diff
@@ -85,8 +85,19 @@ func (r MetricRepository) CreateBatch(
 		lastIters[lastMetric.Key] = lastMetric.LastIter
 	}
 
+	uniqueContexts := make([]*models.Context, 0, len(metrics))
```

Comment:
this var name is a little confusing to me, since this will likely be a lot of not-unique contexts (the repeats are shared). maybe "relatedContexts"?

Reply:
I guess i thought these were not unique because `len(metrics)` -- ie, that you're expecting one context per metric, but they are normalized

Reply:
Okay sounds reasonable